# Analyzing American Teen Birth Rates (Ages 15-19) from 2007 to 2020

## Introduction

Teenage pregnancy has long been a subject of concern and public health interest. Understanding trends in teen birth rates can provide valuable insights into societal changes, health policies, and the well-being of adolescents. In this project, I explored American teen birth rates for the age group 15-19 from 2007 to 2020, aiming to identify any noticeable trends and patterns.

## Project Overview

### Objectives
The primary objectives of this analysis were as follows:
1. **Identify Trends:** Uncover any notable trends or patterns in teen birth rates among adolescents aged 15-19 over the specified timeframe.
2. **Geographic Variations:** Explore variations in teen birth rates across different states to understand regional differences.
3. **Yearly Changes:** Examine how teen birth rates have changed on a yearly basis, highlighting any significant fluctuations or consistent trends.
4. **Insights into Factors:** Attempt to draw insights into potential factors influencing teen birth rates, such as socio-economic conditions, educational programs, and public health policies.

## Expected Outcomes

Through this project, I aimed to provide a comprehensive understanding of the trends in American teen birth rates over the selected years. By uncovering patterns and variations, I aimed to contribute valuable insights to ongoing discussions around adolescent health, education, and public policy.

## Exploring the Data

In [8]:
library(tidyr)

In [9]:
data <- read.csv('/kaggle/input/teen-birthrates/NCHS_-_Teen_Birth_Rates_for_Age_Group_15-19_in_the_United_States_by_County.csv')

In [10]:
head(data)
colnames(data)

Year State   County  State.FIPS.Code County.FIPS.Code Combined.FIPS.Code
1 2003 Alabama Autauga 1               1                1001              
2 2004 Alabama Autauga 1               1                1001              
3 2005 Alabama Autauga 1               1                1001              
4 2006 Alabama Autauga 1               1                1001              
5 2007 Alabama Autauga 1               1                1001              
6 2008 Alabama Autauga 1               1                1001              
  Birth.Rate Lower.Confidence.Limit Upper.Confidence.Limit
1 48.4       42.2                   55.2                  
2 46.4       41.7                   51.5                  
3 44.1       40.3                   48.1                  
4 44.3       40.8                   48.0                  
5 43.7       40.3                   47.2                  
6 41.6       38.4                   45.0

[1] "Year"                   "State"                  "County"                
[4] "State.FIPS.Code"        "County.FIPS.Code"       "Combined.FIPS.Code"    
[7] "Birth.Rate"             "Lower.Confidence.Limit" "Upper.Confidence.Limit"

## A Sum of Teen Birth Rates in Ages 15-19 Across the US (2007-2020)

In [11]:
if (!requireNamespace("dplyr", quietly = TRUE)) {
  install.packages("dplyr")
}
library(dplyr)

In [12]:
sum_birth_rates <- data %>%
  group_by(State) %>%
  summarise(Total_Birth_Rate = sum(Birth.Rate)) %>%
  arrange(desc(Total_Birth_Rate))

# Print the result
print(sum_birth_rates)

# A tibble: 51 × 2
   State          Total_Birth_Rate
   <chr>                     <dbl>
 1 Texas                   234899.
 2 Georgia                 133399.
 3 Kentucky                101818.
 4 Missouri                 79809.
 5 Mississippi              78886.
 6 Tennessee                75052.
 7 Virginia                 72307.
 8 North Carolina           69762.
 9 Arkansas                 69665.
10 Oklahoma                 68073.
# ℹ 41 more rows


The result shows the total birth rates summarized for each state, arranged in descending order. Here are some observations that I have found:

1. **Texas has the highest total birth rate:** Texas leads with a total birth rate of 234,899. This means that, based on the provided data, Texas had the highest number of births among the states.

2. **Southern states dominate the top positions:** States like Georgia, Kentucky, Missouri, Mississippi, and Tennessee follow Texas in the top positions. This suggests a higher concentration of births in the southern region of the United States.

3. **Variation in birth rates among states:** The total birth rates vary significantly among states, indicating differences in population size, demographics, and possibly other factors influencing birth rates.


## A Sum of American Teen Birth Rates in Ages 15-19 Throughout the Years (2007-2020)

In [13]:
sum_birth_rates <- data %>%
  group_by(Year) %>%
  summarise(Total_Birth_Rate = sum(Birth.Rate)) %>%
  arrange(desc(Total_Birth_Rate))

# Print the result
print(sum_birth_rates)

# A tibble: 18 × 2
    Year Total_Birth_Rate
   <int>            <dbl>
 1  2007          147773.
 2  2008          145571.
 3  2006          144637.
 4  2003          141101 
 5  2004          140180.
 6  2009          139463.
 7  2005          138547.
 8  2010          128008.
 9  2011          119573.
10  2012          114237.
11  2013          104736.
12  2014           97268.
13  2015           90814.
14  2016           84066.
15  2017           79129.
16  2018           74495.
17  2019           72410 
18  2020           68095.


The provided results show the total birth rates summarized for each year, arranged in descending order. Here are some observations that I found:

1. **Peak years for total birth rates:** The years 2007 and 2008 had the highest total birth rates, with 147,773 and 145,571 births, respectively. These were peak years for the total number of births among the years covered by the data.

2. **Decline in total birth rates:** There is a general trend of declining total birth rates over the years. The number of births decreased from 141,101 in 2003 to 68,095 in 2020.

4. **Insights for further investigation:** The declining trend may raise questions about factors influencing birth rates, such as economic conditions, social trends, and policy changes. Further investigation could explore these factors and their impact on birth rates over time.

## Conclusion

The analysis of American teen birth rates (ages 15-19) from 2007 to 2020 reveals intriguing insights into the dynamics of adolescent pregnancies. Below are my top overall observations given to me from the analyzed data.

### Declining Trends and Potential Factors

The observed decline in teen birth rates aligns with broader national efforts to address and educate young individuals on reproductive health. Comprehensive sex education, increased access to contraceptives, and awareness campaigns may have contributed to this positive trend.

### Pandemic Influence

It is crucial to acknowledge a limitation in the dataset – the absence of post-pandemic data. The COVID-19 pandemic, with its lockdowns and disruptions to daily life, could have impacted teen birth rates. Increased isolation and changes in social dynamics might have influenced adolescent behaviors, potentially leading to fluctuations in birth rates that we unfortunately have no access to confirm with this data.

### Regional Disparities

Notably, the analysis reveals regional disparities, with Southern states exhibiting higher teen birth rates. This regional trend is a subject of interest and may be attributed to various factors such as differences in cultural norms, socioeconomic conditions, and the effectiveness of sex education programs. Further research is needed to delve into the underlying causes of these disparities.

### Social Acceptance and Abortion

The declining teen birth rates may also be associated with evolving societal attitudes, including greater acceptance of abortion among young adults. The normalization and increased accessibility of reproductive healthcare services could contribute to the choices made by adolescents, impacting birth rates.

---

Understanding the intricacies of teen birth rates is vital for shaping targeted public health policies, educational programs, and support systems. Continuous monitoring, especially in the post-pandemic era, will be crucial to capturing any shifts in trends and addressing emerging challenges.

In conclusion, while the data suggests positive trends in declining teen birth rates, ongoing efforts are necessary to explore regional disparities and adapt strategies to the changing landscape of adolescent reproductive health. The intersection of societal attitudes, healthcare access, and external influences continues to shape the narrative around teen pregnancies in the United States.